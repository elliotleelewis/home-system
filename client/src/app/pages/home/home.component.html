<section class="card bg-primary text-primary-content">
	<div class="card-body">
		<div class="card-title text-xl">Hello.</div>
		<p>I hope you are having a great day!</p>
	</div>
</section>
<section class="hero min-h-[50vh]">
	<div class="hero-content flex-col p-8">
		<i class="fas fa-hand-point-left fa-2x"></i>
		<p>Select a link from the menu...</p>
	</div>
</section>
