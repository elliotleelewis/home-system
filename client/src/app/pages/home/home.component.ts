import { ChangeDetectionStrategy, Component } from '@angular/core';

@Component({
	selector: 'app-home',
	templateUrl: './home.component.html',
	changeDetection: ChangeDetectionStrategy.OnPush,
	standalone: true,
})
export class HomeComponent {}
