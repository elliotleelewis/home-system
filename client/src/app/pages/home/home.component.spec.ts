import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';

import { HomeComponent } from './home.component';

describe('HomeComponent', () => {
	let component: HomeComponent;
	let fixture: ComponentFixture<HomeComponent>;

	beforeEach(
		waitForAsync(() => {
			void TestBed.configureTestingModule({
				declarations: [HomeComponent],
			}).compileComponents();
		}),
	);

	beforeEach(() => {
		fixture = TestBed.createComponent(HomeComponent);
		component = fixture.componentInstance;
		fixture.detectChanges();
	});

	it('should create', () => {
		expect(component).toBeTruthy();
	});

	it('should render', () => {
		expect(fixture).toMatchSnapshot();
	});
});
