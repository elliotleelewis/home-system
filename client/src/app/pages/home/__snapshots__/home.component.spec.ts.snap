// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeComponent should render 1`] = `
<app-home
  __ngContext__={[Function LRootView]}
>
  <section
    class="hero is-info is-bold is-small"
  >
    <div
      class="hero-body"
    >
      <h1
        class="title"
      >
        Hello.
      </h1>
      <h2
        class="subtitle"
      >
        I hope you are having a great day!
      </h2>
    </div>
  </section><section
    class="hero is-medium has-text-centered"
  >
    <div
      class="hero-body"
    >
      <div
        class="title"
      >
        <i
          class="fas fa-hand-point-left fa-2x"
        />
      </div>
      <p
        class="subtitle"
      >
        Select a link from the menu...
      </p>
    </div>
  </section>
</app-home>
`;
